src/utils/index.ts

```diff
@@ -189,12 +208,31 @@ function iso8601ToRippleTime(iso8601: string): number {
 }
 
 export {
-  dropsToXrp,
-  xrpToDrops,
-  toRippledAmount,
-  convertKeysFromSnakeCaseToCamelCase,
-  removeUndefined,
-  rippleTimeToISO8601,
-  iso8601ToRippleTime,
-  isValidSecret
+    computeLedgerHeaderHash,
+    dropsToXrp,
+    xrpToDrops,
+    toRippledAmount,
+    convertKeysFromSnakeCaseToCamelCase,
+    removeUndefined,
+    rippleTimeToISO8601,
+    iso8601ToRippleTime,
```

Comment:
These names should be changed, they aren't very descriptive (see xrpl-py for naming examples)

Reply:
Changed iso8601 to isoTime to make it more intuitive for folks who don't know the acronym (And still keep the unique identifier since iso time is uniquely iso8601)